react-graphql-apollo-tutorial.md

```diff
@@ -1802,7 +1794,7 @@ const Profile = () => (
 );
 {{< /highlight >}}
 
-Furthermore, the boolean can be passed down to the RepositoryList component. There it can be used to show a loading indicator instead of the "More" button. Since the boolean never reaches the RepositoryList component for the initial request, you can be sure that the "More" only changes to the loading indicator when there is a successive request pending.
+The boolean can be passed down to the RepositoryList component. There it can be used to show a loading indicator instead of the `More` button. Since the boolean never reaches the RepositoryList component for the initial request, you can be sure that the `More` only changes to the loading indicator when there is a successive request pending.
```

Comment:
the "More" button

it follows the same rules as The Road to learn React with the "Dismiss" and "More" button, when it only speaks about the label of the button